assets/css/main.css

```diff
@@ -2,623 +2,650 @@ html, body {
-  }
+.block-service{
+  width:25%;
+  position:relative;
+}
+.block-service p{
+    font-size:16px;
+}
+.block-service.right{
+  padding-left:21%;
+  padding-right:4%;;
+}
+.block-service.left{
+  padding-right:21%;
+  padding-left:4%;
 }
 
+.plus{
+  width:20px;
+  height:20px;
+  text-align:center;
+  left:50%;
+  margin-left:-10px;
+  top:95px;
+  position:absolute;
+  height:0;
+  font-size:48px;
+}
 
-@media screen and (max-width: 500px) {
-  .big-splash-inner-wrapper {
-    width: 400px;
-  }
-  .tag-1 {
-    font-size: 1.7rem;
-  }
-  .tag-2 {
-    font-size: 2.1rem;
-  }
-  .tag-button {
-    font-size: 2.1rem;
-  }
+.circle{
+  top:0px;
+  width:180px;
+  height:180px;
+  border-radius:50%;
+  margin:0 auto;
+  position:absolute;
+  border:5px solid #52868e;
+}
+.circle img{
+  position:absolute;
+  width:30%;
+  height:30%;
+  margin-left:-15%;
+  margin-top:34%;
+  opacity:.6;
 }
 
+.block-service.left  .circle{
+  right:35px;
+  top:35px;
+}
+.block-service.right .circle{
+  left:35px;
+  top:35px;
+}
 
-@media screen and (max-width: 400px) {
-  .big-splash-inner-wrapper {
-    width: 100%;
-  }
-  .tag-1 {
-    font-size: 1.5rem;
-  }
-  .tag-2 {
-    font-size: 1.75rem;
-  }
-  .tag-button {
-    font-size: 1.75rem;
-  }
+/*WHO WE ARE*/
+#who-we-are{
+  background:#f5f6f7;
 }
 
-@media screen and (max-width: 350px) {
-  .big-splash-inner-wrapper {
-    width: 100%;
-  }
-  .tag-1 {
-    font-size: 1.25rem;
-  }
-  .tag-2 {
-    font-size: 1.4rem;
-  }
-  .tag-button {
-    font-size: 1.55rem;
-  }
+#who-we-are p{
+  margin-bottom:35px;
 }
 
-.component-wrapper {
-  padding: 0 30px;
+#who-we-are .team-member-blocks{
+    display:flex;
+    justify-content: space-between;
 }
 
-.component-wrapper h2 {
-  font-size: 2.5rem;
-  margin-bottom: 0px;
+#who-we-are #team-wrapper .team-member{
+  width:33.3%;
+  margin:0;
+}
+#who-we-are #team-wrapper .team-member img{
+  width:200px;
+  max-width:100%;
+  border-radius:50%;
 }
 
-.component-wrapper h3 {
-  font-size: 2rem;
-  font-weight: 600;
-  margin: 0 0 10px 0;
+#who-we-are #team-wrapper .team-member h4{
+  margin:15px 0 5px 0;
+  padding:0;
+  color:#52868e;
+  text-transform:uppercase;
+  font-size:22px;
+  letter-spacing:.05em;
 }
+#who-we-are #team-wrapper .team-member h5{
+  margin:0 0 25px 0;
+  padding:0;
+  font-size:16px;
+  letter-spacing:.05em;
+  text-transform:uppercase;
+  color:rgba(24,34,41, 1);
 
-.component-wrapper h4 {
-  font-size: 1.8rem;
-  font-weight: 500;
-  margin: 0;
-  line-height: 0.9;
+}
+#who-we-are #team-wrapper .team-member p{
+  font-size:16px;
+  line-height:24px;
+  margin:0 25px 0;
+  color:#555;
 }
 
-.component-wrapper h5 {
-  font-size: 1.4rem;
-  font-weight: 400;
-  margin: 0 0 5px 0;
-  color: #5F5F5F;
+/*WHAT WE'RE BUILDING*/
+#what-were-building{
+  background:rgba(255,255,255,1);
+}
+#what-were-building .inner-wrapper{
 }
 
-.component-wrapper p {
-  font-weight: 100;
+#what-were-building #project-list{
+    display:flex;
```

Comment:
👏 on all the flexbox usages! Don't forget to do all the browser flags (`-webkit-flex`, etc)